xl/playlist.py

```diff
@@ -2229,22 +2228,21 @@ def load_from_location(self, location):
 
             @return: a list of the playlist names
         """
-        f = None
         for loc in [location, location + ".new"]:
             try:
-                f = open(loc, 'r')
-                break
+                with open(loc, 'r') as fd:
+                    return self.__load_from_location_internal(fd)
             except Exception:
                 pass
-        if f is None:
-            return []
+        return None
+
+    def __load_from_location_internal(self, filedescriptor):
```

Comment:
Considering the rest of the code in this PR, perhaps ```filedescriptor``` should be named ```f``` or ```fd``` for consistency sake?

Reply:
Personally, I don't like short identifiers like `f` because they don't help anything. I've shortened it to `fd` to match the style anyway.